CMakeLists.txt

```diff
@@ -40,40 +40,55 @@ option(AVIF_ENABLE_EXPERIMENTAL_GAIN_MAP
 )
 option(AVIF_ENABLE_EXPERIMENTAL_AVIR "Enable experimental reduced header" OFF)
 
-option(AVIF_CODEC_AOM "Use the AOM codec for encoding/decoding (see AVIF_CODEC_AOM_DECODE/AVIF_CODEC_AOM_ENCODE)" OFF)
-option(AVIF_CODEC_DAV1D "Use the dav1d codec for decoding" OFF)
-option(AVIF_CODEC_LIBGAV1 "Use the libgav1 codec for decoding" OFF)
-option(AVIF_CODEC_RAV1E "Use the rav1e codec for encoding" OFF)
-option(AVIF_CODEC_SVT "Use the SVT-AV1 codec for encoding" OFF)
-option(AVIF_CODEC_AVM "Use the AVM (AV2) codec for encoding/decoding (EXPERIMENTAL)" OFF)
+macro(set_local_or_system_option VAR DEFAULT TEXT)
+    # Deal with the older way of setting options.
+    if(DEFINED AVIF_LOCAL_${VAR})
+        set(DEFAULT ${AVIF_LOCAL_${VAR}})
+    elseif(DEFINED AVIF_${VAR})
+        set(DEFAULT ${AVIF_${VAR}})
+    endif()
+    set(AVIF_${VAR} ${DEFAULT} CACHE STRING ${TEXT} FORCE)
+    set_property(CACHE AVIF_${VAR} PROPERTY STRINGS OFF LOCAL SYSTEM)
+endmacro()
+macro(set_codec_option CODEC NAME ENCDEC EXTRA)
+    # Deal with the older way of setting options.
+    if(DEFINED AVIF_CODEC_${CODEC})
+        set(AOM_CODEC_DEFAULT ${AVIF_CODEC_${CODEC}})
```

Comment:
Since this is used by more than aom, I'm guessing that this variable name is a mistake or got held over from an earlier version of this macro?